README.Rmd

```diff
@@ -113,99 +113,146 @@ retro_nat_germany <- nat_germany_hosp |>
 retro_nat_germany
 ```
 
+This data is already in a format that can be used with `epinowcast`, as it contains
+
+- a reference date (column `reference_date`): the date of the observation, in this example the date of a positive test
+- a report date (column `report_date`): the date of report for a given set of observations by reference date
+- a count (column `confirm`): the total (i.e. cumulative) number of hospitalisations by reference date and report date
+
```

Comment:
I tried adding more structure that is easy to scan over

Reply:
Like it.